### ISSUE TITLE
GridspecLayout does not work with the same view's model?

### ISSUE THREAD

**Issue by hainm** [jupyter-widgets/ipywidgets#2509]
Hi, please see below example. Is it intentional or a bug? (The GridBox can have multiple views of the same model). Thanks.

![grid](https://user-images.githubusercontent.com/4451957/61553560-a549c600-aa28-11e9-9c75-b5c773e4f1cf.png)

**Comment by jasongrout** [comment]
Looks like a bug to me

**Comment by hainm** [comment]
yeah, it appears that two views of the same model are put in the same parent in the right.
